### ISSUE TITLE
Upgrade typescript version

### ISSUE THREAD

**Issue by alexo** [wro4j/wro4j#987]
Use latest version of typescript & use webjar.

**Comment by natami** [comment]
What is the status of this change? Is it in 1.8.0 release?

**Comment by alexo** [comment]
@natami no, it was moved to candidate milestone, which is used as a pool for next tasks to start to work on... There is no clear timeline... but if you need it urgently and willing to contribute, you're welcome..

**Comment by totof3110** [comment]
+1. This would be a great feature! :)

**Comment by trajano** [comment]
I am thinking it is a matter of changing https://github.com/wro4j/wro4j/blob/b09a44747011006578d3a916fa464a39f633e860/wro4j-extensions/src/main/java/ro/isdc/wro/extensions/processor/support/typescript/TypeScriptCompiler.java and making it reference the webjar versions.

**Comment by alexo** [comment]
@trajano great idea. Your contribution would be greatly appreciated.
